frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -24,20 +25,25 @@ export const getSelectOverrides = (): MantineThemeOverride["components"] => ({
         color: "text-dark",
       },
     }),
-    styles: (theme, _, { size = "md" }) => ({
-      ...getSelectInputOverrides(theme),
-      ...getSelectItemsOverrides(theme, size),
-      // For epic (metabase#38699)
-      dropdown: {
-        background: "var(--mb-color-background)",
-        borderColor: "var(--mb-color-border)",
-        ">div": {
-          maxHeight: "none !important",
-        },
-        padding: 0,
-        overflow: "auto",
-      },
-    }),
+    styles: getSelectStylesOverrides,
+  },
+});
+export const getSelectStylesOverrides = (
```

Comment:
I reuse this in `default-component-theme`, so I can remove the `maxHeight`. I think this is a better solution since we don't have to have any `isEmbeddingSdk` checks, and I don't think we need this for the SDK